modules/cert-manager-remove-resources-console.adoc

```diff
@@ -17,7 +17,26 @@ Optionally, after uninstalling the {cert-manager-operator}, you can remove its r
 
 . Log in to the {product-title} web console.
 
-. Remove CRDs that were installed by the {cert-manager-operator}:
+. Remove the deployments of the cert-manager components such as cert-manager, CA injector and Webhook present in the `cert-manager` namespace.
+
+.. Click the *Project* drop-down menu to see a list of all available projects. Select the *cert-manager* project.
+
+.. Navigate to *Workloads* -> *Deployments*.
+
+.. Click the deployment that you want to delete.
+
+.. Click the *Actions* drop-down menu, and select *Delete Deployment* to see a confirmation dialog box.
+
+.. Click *Delete* to delete the deployment.
+
+. Delete deployments of the cert-manager components such as `cert-manager`, `cainjector` and `webhook` present in the `cert-manager` namespace by using the command-line interface (CLI).
```

Comment:
If previous `Remove the deployments of the cert-manager components ...` in web console is done, then it already does what `Delete deployments of the cert-manager components ...` by CLI will do. By console and by CLI are equivalent.
```suggestion
. Alternatively, delete deployments of the cert-manager components such as `cert-manager`, `cainjector` and `webhook` present in the `cert-manager` namespace by using the command-line interface (CLI).
```